unlock-app/src/utils/durations.js

```diff
@@ -63,3 +65,18 @@ export function durationsAsTextFromSeconds(seconds) {
 export function secondsAsDays(seconds) {
   return Math.ceil(seconds / 86400).toString()
 }
+
+/**
+ * Given an epoch timestamp, returns a string of the form 'Month Day, Year' (eg, 'Dec 31, 1980')
+ * @param timestamp
+ * @returns {string}
+ */
+export function expirationAsDate(timestamp) {
```

Comment:
Could you add a TODO to make this a little bit smarter to include hours/minutes when the lock is about to expire?

Reply:
I went ahead and just implemented this.